### ISSUE TITLE
Cut connectors misplaced

### ISSUE THREAD

**Issue by jeffdemeutter** [SoftFever/OrcaSlicer#7583]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.1.1
and
2.2.0

### Operating System (OS)

Windows

### OS Version

Windows 11

### Additional system information

13th gen intel core i7-1360P
16gb ddr4
nvidia RTX A500

### Printer

Bambu p1s

### How to reproduce

open Cut panel 
Set Cut height to 22mm
open `edit connectors`
select snap connectors with the following settings 
![image](https://github.com/user-attachments/assets/a1638647-175a-4d15-ab47-97bde5a6404b)
confirm the connectors
perform the cut
click yes on the `non-manifold edges` popup
![image](https://github.com/user-attachments/assets/6b60fdf7-23e0-462a-a443-0cd294300455)

### Actual results

![image](https://github.com/user-attachments/assets/fadae51f-ce79-495a-a311-6df898e819a5)
![image](https://github.com/user-attachments/assets/b141fdb0-4d5c-4812-bcb6-4850e4c8f5b2)

### Expected results

![image](https://github.com/user-attachments/assets/7ba90ea6-8716-46fb-93b4-7b878a92aa0c)
![image](https://github.com/user-attachments/assets/90404203-2847-4b58-9fca-74479694134a)

### Project file & Debug log uploads

[connectorIssue.zip](https://github.com/user-attachments/files/17916387/connectorIssue.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

### Anything else?

In one part of the object it seems to be placed fine and correct, but on the other object it gets offset.

I have also checked in BambuStudio, and it results in exactly the same.

I know that the object is currently too big for my build plate, but I first want to split the object as i am trying now, so I can cut it later into piececs that fit on the build plate.

**Comment by Noisyfox** [comment]
I think the problem here is the model size changed after the repair, which lead to the offset of the connectors.
If you don't fix the model, you will notice this part:
![image](https://github.com/user-attachments/assets/97032664-d901-4251-be9d-9143c80b4d3f)
(Note how the connectors are placed correctly)

However once you fix the model:
![image](https://github.com/user-attachments/assets/54ddf831-3cb7-4625-a02e-d172d8dedb97)

**Comment by Noisyfox** [comment]
What you need to do is place the model perfectly flat on the plate, by using:
![image](https://github.com/user-attachments/assets/13626c5d-d8f5-4a76-89ff-1a9af6d7825f)

Then you can cut it perfectly without the need of model fix
![image](https://github.com/user-attachments/assets/4959cbff-1a18-4aba-9e4b-f008f5a26788)

**Comment by jeffdemeutter** [comment]
I already was using lay on face, but I was laying it on the letters. I just tried to lay it on the background and cut it from there. This worked! 

I have made this model myself, and can ensure that the faces on the XY plane are all perfectly parallel. So when I lay it down on the letters, I would expect it to behave the exact same, no?

**Comment by Noisyfox** [comment]
Weird... I lay on the latters too and it works somehow.

**Comment by jeffdemeutter** [comment]
yes very odd, I just can't get it to work when its laid flat on the letters side. I tried multiple orientation around Z axis after. I tried flipping the connectors. Nothing seems to make it work for me.
